biggus/__init__.py

```diff
@@ -561,6 +563,175 @@ def _normalise_keys(self, keys):
         return keys
 
 
+class ArrayContainer(Array):
```

Comment:
> A separate PR?

> Although arguably that horse has bolted.

I'm not sure I follow? I didn't want to make changes to the TransposedArray object in this PR, but I envisage making the change once an ArrayContainer exists in biggus.